neo.UnitTests/IO/Caching/UT_FifoSet.cs

```diff
@@ -2,7 +2,7 @@
 using Neo.IO.Caching;
 using System.Linq;
 
-namespace Neo.UnitTests
+namespace Neo.UnitTests.IO.Caching
```

Comment:
Some folder were better organized, right?

Reply:
I believe the UTs gonna be more clear, looks good to me.
Also for filtering tests it will be easier.

Reply:
OK! we can make more folder and change code's location and namespace in next UT pull request  which is coming soon.